composer.json

```diff
@@ -33,7 +33,7 @@
         }
     },
     "require": {
-        "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
Can you also drop support for PHP 7.4, changing `config.platform.php` to `8.0.99`, and run a composer update to regenerate the lock file with latest deps. `composer bump --dev-only` would also help.